docsite/docs/Reference/fal-scripts-path.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 1
+---
+
+# Variable `fal-scripts-path`
+
+fal uses the `fal-scripts-path` [dbt variable](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables#defining-variables-in-dbt_projectyml) to scripts in your project.
+
+Let's consider the following file structure for some examples:
+```
+.
+├── dbt_project.yml
+├── models
+│   ├── schema.yml
+│   └── some_model.sql
+└── scripts
+    ├── after.py
+    ├── before.py
+    └── utils
+        ├── my_utils.py
+        └── process
+            └── process_df.py
+```
+
+By default `fal-scripts-path` is the [dbt project directory](https://docs.getdbt.com/reference/dbt_project.yml) (where the `dbt_project.yml` is located). 
+
+But it can be chagend in the [vars](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables#defining-variables-in-dbt_projectyml). An example of setting the variable would be:
```

Comment:
```suggestion
But it can be changed in the [vars](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables#defining-variables-in-dbt_projectyml) variable. An example of setting the variable would be:
```